packages/web_benchmarks/lib/src/recorder.dart

```diff
@@ -1040,18 +1037,19 @@ class _RecordingWidgetsBinding extends BindingBase
     _instance = this;
   }
 
-  static _RecordingWidgetsBinding get instance => _instance;
-  static _RecordingWidgetsBinding _instance;
+  static _RecordingWidgetsBinding get instance =>
+      BindingBase.checkInstance(_instance);
+  static _RecordingWidgetsBinding? _instance;
 
   /// Makes an instance of [_RecordingWidgetsBinding] the current binding.
   static _RecordingWidgetsBinding ensureInitialized() {
-    if (_RecordingWidgetsBinding.instance == null) {
+    if (_instance == null) {
       _RecordingWidgetsBinding();
     }
     return _RecordingWidgetsBinding.instance;
   }
 
-  FrameRecorder _recorder;
+  FrameRecorder? _recorder;
```

Comment:
Similar to my earlier comment, this needs a comment explaining when and why it's safe to force-unwrap.